ui/.eslintrc.base.js

```diff
@@ -13,7 +13,13 @@
 
 // Base eslint configuration for typescript projects
 module.exports = {
-  extends: ['plugin:@typescript-eslint/recommended', 'plugin:prettier/recommended'],
+  extends: [
```

Comment:
We used to have two "base" configs for eslint while we really only needed one, so I just condensed them down early on into a single config. (Sorry about the additional noise in this PR).